.github/workflows/ci-rust.yml

```diff
@@ -138,6 +138,9 @@ jobs:
         env:
           RUST_LOG: debug
 
+      - name: unlock keyring
+        uses: t1m0thyj/unlock-keyring@v1
```

Comment:
pin the version (e.g. `uses: taiki-e/install-action@b7add58e53e52e624966da65007ce24524f3dcf3 # pin v2.27.9`)

Reply:
Done